tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "skipLibCheck": true
   },
-  "include": ["**/*", ".eslintrc.cjs"],
+  "include": ["**/*", "**/*.astro", ".eslintrc.cjs"],
```

Comment:
An attempt at making the ESLint server stop being weird. Didn't work, but this is correct anyway